db/schema.rb

```diff
@@ -1202,6 +1202,48 @@
   add_index "weams", ["institution"], name: "index_weams_on_institution", using: :btree
   add_index "weams", ["state"], name: "index_weams_on_state", using: :btree
 
+  create_table "yellow_ribbons", force: :cascade do |t|
+    t.string   "facility_code"
+    t.string   "school_name_in_yr_database"
+    t.string   "school_name_in_weams"
+    t.string   "campus"
+    t.string   "street_address"
+    t.string   "city"
+    t.string   "state"
+    t.string   "zip"
+    t.string   "public_private"
```

Comment:
Would like some more information on this column, seeing the following values:

* `- Please make a selection -`
* `NULL`
* `Private`
* `Private NP`
* `Private-NP`
* `Public`
* ``